theme/templates/includes/author-preview.html

```diff
@@ -13,7 +13,7 @@ <h4 class="modal-title">Author</h4>
                             <tr v-if="is_person">
                                 <td class="text-center"><i class="fa fa-user" aria-hidden="true"></i></td>
                                 <th>Name: </th>
-                                <td>${author.name}</td>
+                                <td>${author.name == ', ' ? '' : author.name}</td>
                             </tr>
```

Comment:
not a big deal, especially since code outside of Discover doesn't follow strong typing or linting, but JS avoids type coercion with triple relational operators, in this case === or as another example of non-equality !==

Reply:
@cuahsimarko Thanks for pointing it out. I'll keep this in mind when I revamp the backend code for this. I may not need this frontend change after all with the backend code changes.

Reply:
For when you do the backend PR, if it is useful for you to compare approaches, you can see how new Discover handles the various data inconsistencies (as temporary workarounds to obfuscate the problem from users). It's lines 200-236 https://github.com/hydroshare/hydroshare/blob/5ef2af2497b29862e11d4d0ed09cbabec6f79664/hs_discover/views.py#L200

Reply:
Sure. Thanks for the pointer @cuahsimarko